package.json

```diff
@@ -127,7 +126,9 @@
     "redux-mock-store": "1.5.3",
     "snyk": "^1.161.1",
     "source-map-explorer": "1.8.0",
+    "tslint": "^5.17.0",
```

Comment:
Can we get rid of the `^` and pin it to specific versions?
Snyk is the only one that should have `^`.
`tslint-react` also has `^`